bot/code_review_bot/workflow.py

```diff
@@ -191,6 +197,41 @@ def publish(self, revision, issues, task_failures, links):
         )
         stats.add_metric("analysis.issues.publishable", nb_publishable)
 
+        if self.lando is not None:
+            logger.info(
+                "Publishing warnings to lando for {0} errors and {1} warnings".format(
+                    nb_publishable_errors, nb_publishable - nb_publishable_errors
+                ),
+                revision=revision.id,
+                diff=revision.diff["id"],
+            )
+            self.lando.del_all_warnings(revision.id, revision.diff["id"])
+
+            # Determine if we need to post something or not
+            if nb_publishable > 0:
+                lando_warning_message = (
```

Comment:
Nit: Let's define the template at the top of the file, so it's more readable (like https://github.com/mozilla/code-review/blob/ef9459f2211cd9eca869e5ea60faecb508288bc1/bot/code_review_bot/report/mail.py#L17).

We should also make it clear as part of the message that errors will likely lead to a back-out.
We could write something like:
`The code review bot found XXX errors which should be fixed to avoid backout and XXX warnings`